CHANGELOG.md

```diff
@@ -14,6 +15,150 @@
+- URL Rewrites and Path redirects have been added as new "Experimental" features
+  [#945](https://github.com/kubernetes-sigs/gateway-api/pull/945)
+
+### Bug Fixes
+
+- Fixes a problem that would cause webhook deployment to fail on Kubernetes
+  v1.22 and greater.
+  [#991](https://github.com/kubernetes-sigs/gateway-api/pull/991)
+- Fixes a bug where the `Namespace` could be unspecified in `ReferencePolicy`
+  [#964](https://github.com/kubernetes-sigs/gateway-api/pull/964)
+- Fixes a bug where v1alpha2 GatewayClass controller names were not being
+  shown in the output of `kubectl get gatewayclasses`
+  [#909](https://github.com/kubernetes-sigs/gateway-api/pull/909)
+
+### General Improvements
+
+- Conformance tests were introduced with [GEP-917][gep-917] and multiple
+  conformance tests were added from a variety of contributors under the
+  `conformance/` directory.
+- The status of the GatewayClass "Accepted" condition for the `GatewayClass`
+  is now present in `kubectl get` output.
+  [#1168](https://github.com/kubernetes-sigs/gateway-api/pull/1168)
+- New `RouteConditionReason` types `RouteReasonNotAllowedByListeners` and
+  `RouteReasonNoMatchingListenerHostname` were added.
+  [#1155](https://github.com/kubernetes-sigs/gateway-api/pull/1155)
+- New `RouteConditionReason` type added with `RouteReasonAccepted`,
+  `RouteReasonResolvedRefs` and `RouteReasonRefNotPermitted` constants.
+  [#1114](https://github.com/kubernetes-sigs/gateway-api/pull/1114)
+- Introduced PreciseHostname which prevents wildcard characters in relevant
+  Hostname values.
+  [#956](https://github.com/kubernetes-sigs/gateway-api/pull/956)
+
+[gep-917]:https://gateway-api.sigs.k8s.io/geps/gep-917/
+
+### Validation Improvements
+
+- Webhook validation now ensures that a path match exists when required by path
+  modifier in filter.
+  [#1171](https://github.com/kubernetes-sigs/gateway-api/pull/1171)
+- Webhook validation was added to ensure that only type-appropriate fields are
+  set in `HTTPPathModifier`.
+  [#1124](https://github.com/kubernetes-sigs/gateway-api/pull/1124)
+- The Gateway API webhook is now deployed in a `gateway-system` namespace
+  instead of `gateway-api`.
+  [#1051](https://github.com/kubernetes-sigs/gateway-api/pull/1051)
+
+### Breaking Changes
+
+- The v1alpha1 API version was deprecated and removed.
+  [#1197](https://github.com/kubernetes-sigs/gateway-api/pull/1197)
+  [#906](https://github.com/kubernetes-sigs/gateway-api/issues/906)
+- The `NamedAddress` value for `Gateway`'s `spec.addresses[].type` field has
+  been deprecated, and support for domain-prefixed values (like
+  `example.com/NamedAddress`) has been added instead to better represent the
+  custom nature of this support.
+  [#1178](https://github.com/kubernetes-sigs/gateway-api/pull/1178)
+- Implementations are now expected to use `404` instead of `503` responses when
+  the data-plane has no matching route.
+  [#1151](https://github.com/kubernetes-sigs/gateway-api/pull/1151)
```

Comment:
Mind updating this with the assumption that #1200 is resolved before we cut this release?

Reply:
:+1: 

```suggestion
- Implementations are now expected to use `502` instead of `503` responses when
  the data-plane has no matching route.
  [#1151](https://github.com/kubernetes-sigs/gateway-api/pull/1151)
```